File: src/nl/rubensten/texifyidea/LatexCommenter.java
package nl.rubensten.texifyidea;

import com.intellij.lang.Commenter;
import org.jetbrains.annotations.Nullable;

/**
 * @author Sten Wessel
 */
public class LatexCommenter implements Commenter {

    @Nullable
    @Override
    public String getLineCommentPrefix() {
        return "%";
    }

    @Nullable
    @Override
    public String getBlockCommentPrefix() {
        return null;
    }

    @Nullable
    @Override
    public String getBlockCommentSuffix() {
        return null;
    }

    @Nullable
    @Override
    public String getCommentedBlockCommentPrefix() {
        return null;
    }

    @Nullable
    @Override
    public String getCommentedBlockCommentSuffix() {
        return null;
    }
}

File: src/nl/rubensten/texifyidea/editor/LatexCommenter.kt
package nl.rubensten.texifyidea.editor

import com.intellij.lang.Commenter

/**
 * @author Sten Wessel
 */
open class LatexCommenter : Commenter {

    override fun getLineCommentPrefix() = "%"

    override fun getBlockCommentPrefix() = ""

    override fun getBlockCommentSuffix() = ""

    override fun getCommentedBlockCommentPrefix() = ""

    override fun getCommentedBlockCommentSuffix() = ""
}
